samples/addons/jaeger.yaml

```diff
@@ -68,6 +68,10 @@ spec:
       port: 80
       protocol: TCP
       targetPort: 16686
+    - name: http2-query
```

Comment:
Would `grpc-query` perhaps be more appropriate?

Reply:
@jpkrohling I will change this to say 

```yaml
    # Note: Change port name if you add '--query.grpc.tls.enabled=true'
    - name: grpc-query
```

For the kinds of things I am doing -- `kubectl port-forward`-ing and using _grpcurl_ from my laptop the port name won't matter.  Meshed pods should be able to query Jaeger (e.g. if I want to build a tool that uses Jaeger's jaeger.api_v2.QueryService gRPC service).  See https://github.com/istio/istio/issues/16282#issuecomment-665349098 for why the name can't start with `grpc` if gRPC is using TLS.

Reply:
Not sure I understand. `http2` and `grpc` should be the same, if not its a bug. Let me know if its not working with either

Reply:
The linked issue is about using TLS with a port named "grpc". TLS with a port named "http2" is equally invalid